CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.15.0 Release
+- Release Date: December 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Highlights
+ - Added mention and reply mention highlighting.
+ - Added a sliding animation for the reaction list.
+ - Added support for pinned posts.
+ - Added support for jumbo emojis.
+ - Added support for interactive dialogs.
+ - Improved UI for the long press menu and emoji reaction viewer
+
+### Improvements
+ - Added the ability to include custom headers with requests.
+ - Added support for clearing notifications from a specific channel.
```

Comment:
I wouldn't say support, this should be a statement: Push Notifications are grouped by channels are cleared once the channel is read, or something like that